api/transport/errors.go

```diff
@@ -49,3 +51,17 @@ func IsTimeoutError(err error) bool {
 	_, ok := err.(errors.TimeoutError)
 	return ok
 }
+
+// UnrecognizedProcedureError returns an error for the given request,
+// such that IsUnrecognizedProcedureError can distinguish it from other errors
+// coming out of router.Choose.
+func UnrecognizedProcedureError(req *Request) error {
+	return errors.RouterUnrecognizedProcedureError(req.Service, req.Procedure)
+}
+
+// IsUnrecognizedProcedureError returns true if the request could not be processed
+// because the receiving service did not handle the procedure.
+func IsUnrecognizedProcedureError(err error) bool {
```

Comment:
How about,

> [..] returns true on errors returned by Router.Choose if the request cannot be handled because the service does not know how to handle the procedure.